sandpack-react/src/components/CodeEditor/CodeMirror.tsx

```diff
@@ -144,7 +134,10 @@ export const CodeMirror = React.forwardRef<CodeMirrorRef, CodeMirrorProps>(
     );
 
     const c = useClasser(THEME_PREFIX);
-    const { listen } = useSandpack();
+    const {
+      listen,
+      sandpack: { additionalLanguages },
```

Comment:
I believe we can get this value from this component `props`, instead of the context. The additional language syntax doesn't need to be widely shared between sandpack instances, as this is a very particular implementation of CodeMirror.